Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleGeoTrackingSceneView.swift

```diff
@@ -137,21 +140,32 @@ public struct WorldScaleGeoTrackingSceneView: View {
         .task {
             do {
                 try await locationDataSource.start()
-                await withTaskGroup(of: Void.self) { group in
-                    group.addTask {
-                        for await location in locationDataSource.locations {
-                            lastLocationTimestamp = location.timestamp
-                            currentLocation = location
-                            await updateSceneView(for: location)
-                        }
-                    }
+                for await location in locationDataSource.locations {
+                    lastLocationTimestamp = location.timestamp
+                    currentLocation = location
+                    updateSceneView(for: location)
                 }
             } catch {}
         }
-        .toolbar {
-            ToolbarItem(placement: .bottomBar) {
-                if !calibrationViewIsHidden {
-                    calibrationView
+        .overlay(alignment: calibrationViewAlignment) {
```

Comment:
Ideally, both the button and the calibration view are placed within the safe area. Currently the whole safe area situation is kind of messy, which can be fixed in a separate PR if you like.

- The AR view and coaching overlay should ignore safe area on all edges
- The status view is in safe area at top edge
- The calibration view and button are in safe area at each alignment

Reply:
The safe area issues can be addressed in a separate PR.